runtest.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# fail if smth fails
+# the whole env will be running if test suite fails so you can debug
+set -e
+
+# for debugging this script, b/c I sometimes get
+# unable to prepare context: The Dockerfile (Dockerfile.tests) must be within the build context (.)
+set -x
+
+# test coverage threshold
+COVERAGE_THRESHOLD=15
```

Comment:
Hey @samuzzal-choudhury actually had a discussion with @tisnik and since we have to mock everything in the tests including BigQuery and S3 storage, the actual functions mostly are not being used since otherwise it would involve calling BQ and S3 actually + they are from rudra. So he told me then lower coverage is ok, let me know what you think.